app/models/image.rb

```diff
@@ -0,0 +1,12 @@
+class Image < ActiveRecord::Base
+  has_many :comments,
+        -> { extending WithUserAssociationExtension },
+        dependent: :destroy
+
+  has_many :likes, dependent: :destroy
+  belongs_to :user
+  has_attached_file :image, :styles => { :medium => "300x300>", :thumb => "100x100>" }, :default_url => "/images/:style/missing.png"
```

Comment:
Metrics/LineLength: Line is too long. [132/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)